### ISSUE TITLE
Cmake find library issue

### ISSUE THREAD

**Issue by nerdralph** [krrishnarraj/clpeak#49]
Cmake doesn't find OpenCL with AMDGPU-pro.
As a hack/workaround I added the lib path to the find_library hints:
/opt/amdgpu-pro/lib/x86_64-linux-gnu/

I came across lots of bug reports about Cmake multiarch issues, but it's not clear what the "correct" solution is.

**Comment by nerdralph** [comment]
I tried find_package(OpenCL), but that didn't work.  I'm not a cmake fan, so it's still possible it could work with more changes.  I'm using cmake 3.5.1 on Ubuntu 16.04.

**Comment by hjmallon** [comment]
Yeah. Sorry I haven't used AMDGPU-pro. Seems that path isn't included in the OpenCL module in cmake. https://gitlab.kitware.com/cmake/cmake/blob/master/Modules/FindOpenCL.cmake

**Comment by krrishnarraj** [comment]
Must be fixed by now
